rtgui/threadutils.cc

```diff
@@ -17,6 +17,7 @@
  *  along with RawTherapee.  If not, see <https://www.gnu.org/licenses/>.
  */
 #include "threadutils.h"
+#include <glibmm/threads.h>
```

Comment:
this shouldn't be added if you remove the `Glib::Threads` (here it builds without)